indra/newview/llvoicechannel.cpp

```diff
@@ -768,6 +774,11 @@ LLVoiceChannelP2P::LLVoiceChannelP2P(const LLUUID      &session_id,
     mReceivedCall(false),
     mOutgoingCallInterface(outgoing_call_interface)
 {
+    std::string sip_uri = LLVoiceClient::getInstance()->sipURIFromID(other_user_id);
```

Comment:
This works, but the intent is to keep mChannelInfo opaque as far as anything outside of llvoiceclient.cpp and the voice providers. 

Would probably be better to change sipURIFromID into a different call which returns a full LLSD mChannelInfo blob (including voice server type, etc.)

Reply:
The problem is: there are no full credencials yet so a 'full' blob doesn't appear to be an option. I will double check that, but I likely will go with some kind of `LLVoiceClient::getInstance()->getTemplateChannelInfo()`

Reply:
That works for me.